.circleci/config.yml

```diff
@@ -486,7 +486,11 @@ workflows:
 
   build_and_test:
     jobs:
+      - approval:
+          type: approval
```

Comment:
Just to check – is this approval for any build (including PR branches before merge?)

Reply:
Yes this is approval for any build including PR's as we want to be able to run master without interruption, so we need to be able to control all PR's and there builds, while we wait on a licence.